setup/src/Magento/Setup/Module/Di/Code/Generator/InterceptionConfigurationBuilder.php

```diff
@@ -10,9 +10,14 @@
 use Magento\Framework\App\Area;
 use Magento\Framework\App\Cache\Manager;
 use Magento\Framework\App\Interception\Cache\CompiledConfig;
+use Magento\Framework\App\ObjectManager;
 use Magento\Framework\Interception\Config\Config as InterceptionConfig;
-use Magento\Setup\Module\Di\Code\Reader\Type;
+use Magento\Framework\Interception\Definition\Runtime;
+use Magento\Framework\Interception\ObjectManager\ConfigInterface;
 use Magento\Framework\ObjectManager\InterceptableValidator;
+use Magento\Setup\Module\Di\Code\Reader\Type;
+use function array_merge;
+use function array_unique;
```

Comment:
I'm not sure that it's really needed here

Reply:
```suggestion
```

Reply:
Fixed

Reply:
These probably were added for increased performance.
I'm going to link just [one blog](https://www.bram.us/2020/04/18/php-performance-tip-use-fully-qualified-function-calls-when-working-with-namespaces/), but there is plenty of information about this trick available on the web.

Not sure how much it will matter in this particular case though ...

Reply:
@hostep I don't see any benchmark results for language constructions.
It's interested and I'll check it when I have time

Reply:
The easiest way to test it would probably be to install a Magento version using composer and also install the following composer plugin: https://github.com/dg/composer-backslasher and then check performance differences somehow.
I have no experience with this plugin and I don't know if it will work properly on all Magento modules and their dependencies. But it's probably worth taking a shot.

Anyways, this is completely off topic in this PR 😆